src/ReactApp.js

```diff
@@ -1,12 +1,12 @@
-/* eslint-disable linebreak-style */
-/* eslint-disable react/jsx-indent */
-/* eslint-disable linebreak-style */
 import './App.css';
+import MyCalculator from './components/scientificcalculator/calculate';
 
 function App() {
-// eslint-disable-next-line indent
-return (
-<h1>This is my First React Application</h1>);
+  return (
+    <>
+      <MyCalculator />
+    </>
+  );
 }
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you rename the `src/ReactApp.js` file `src/App.js` for good react naming rules.